tcp-ballerina/build.gradle

```diff
@@ -281,6 +281,8 @@ publishing {
             credentials {
                 username = System.getenv("packageUser")
                 password = System.getenv("packagePAT")
+                username = System.getenv("publishUser")
```

Comment:
@MohamedSabthar I think lines 282, 283 should be removed.

Reply:
+1, fixed in #177